manifests/init.pp

```diff
@@ -137,7 +134,7 @@
   # Update the reports setting in puppet.conf
   ini_subsetting { 'puppetserver puppetconf add servicenow report processor':
     ensure               => present,
-    path                 => "${puppet_base}/puppet.conf",
+    path                 => "${settings::confdir}/puppet.conf",
```

Comment:
Hmm, you sure this is right? Asking b/c normally you'd specify settings in `puppet.conf`, but `confdir` (a setting) is used here as `puppet.conf`'s location (chicken vs egg problem).

Reply:
Interestingly when I looked this up, the documentation says that not only the config file directory, but the config file name itself, can also be set, but only via the commandline. Once I start reading the documentation though, I wonder if instead of using `"${settings::confdir}/puppet.conf"`, maybe we should be using `"${settings::config}"  instead? https://puppet.com/docs/puppet/5.5/configuration.html#confdir

Reply:
I gave settings::config a go and I got this error.
`error during compilation: Parameter path failed on Ini_subsetting[puppetserver puppetconf add servicenow report processor]: File paths must be fully qualified, not 'settings::config'`

Reply:
oh nvmd it works, just had to interpolate it in a string like so,
`"${settings::config}"`

Reply:
@ekinanp Please do chime in with whether you like this change or not. I won't merge at least until you do.

Reply:
Looking at https://puppet.com/docs/puppet/5.5/configuration.html#config, seems like `settings::config` defaults to `<confdir>/<config_file_name>` so this should be fine.